civis/utils/_jobs.py

```diff
@@ -26,7 +30,41 @@ def run_job(job_id, api_key=None, client=None):
     if client is None:
         client = APIClient(api_key=api_key, resources='all')
     run = client.jobs.post_runs(job_id)
-    return CivisFuture(client.jobs.get_runs,
-                       (job_id, run['id']),
-                       client=client,
-                       poll_on_creation=False)
+    fut = CivisFuture(client.jobs.get_runs,
+                      (job_id, run['id']),
+                      client=client,
+                      poll_on_creation=False)
+    if wait:
+        fut.result()
+    return fut
```

Comment:
Hmm.  On second thought, I'm warming up to this approach.  Returning a future in a "done" state seems pretty reasonable, and adding sugar to CivisFuture like `block()` will be confusing because it is just `.result()` that returns `None`.  I do generally want to avoid adding the parameter `wait` to the higher level functions in preference of suggests chaining `.result()` instead.